web/utils.py

```
@@ -938,3 +942,75 @@ def geocode_address(address):
     except Exception as e:
         logger.error(f"Geocoding error: {e}")
         return None
+
+
+def notify_capacity_thresholds(
+    teacher, 
```

Ruff (0.8.2)

Missing type annotation for function argument `teacher`

(ANN001)